scalalib/src/mill/scalalib/JavaModule.scala

```diff
@@ -1006,6 +1006,11 @@ trait JavaModule
 
   def forkWorkingDir: T[Path] = T { T.workspace }
 
+  /**
+   * Files extensions that need to be managed by Zinc together with class files
```

Comment:
This means essentially, zinc will remove them (but not create to rename/move them) when a equally named class files is about to be removed, right?

```suggestion
   * Files extensions that need to be managed by Zinc together with class files.
   * This means, if zinc needs to remove a class file, it will also remove files
   * which match the class file basename and a listed file extensions.
```

Reply:
@lolgab Looks like this got lost during your last force-push.

Reply:
Yeah, messed up with the branches. Pushed it again.